application/src/main/webapp/vue-app/components/ExoChatContactList.vue

```diff
@@ -95,7 +95,11 @@
               v-if="mq === 'mobile'"
               :class="{'is-fav': contact.isFavorite}"
               class="uiIcon favorite"></div>
-            <div v-if="mq === 'mobile' || drawerStatus" :class="[drawerStatus ? 'last-message-time-drawer last-message-time' : 'last-message-time']">{{ getLastMessageTime(contact) }}</div>
+            <div
+              v-if="mq === 'mobile' || drawerStatus"
```

Comment:
This check ``` mq === 'mobile' ``` is supposed to indicate if we are in a responsive mode , doesn't it work ?